js_modules/dagit/packages/core/src/runs/LogsProvider.tsx

```diff
@@ -75,20 +77,24 @@ const reducer = (state: State, action: Action) => {
         clientsideKey: `csk${idx}`,
       }));
       return {...state, nodes, loading: action.hasMore};
+    case 'set-cursor':
+      return {...state, cursor: state.nodes.length - 1};
     case 'reset':
-      return {nodes: [], loading: true};
+      return {nodes: [], cursor: -1, loading: true};
     default:
       return state;
   }
 };
 
 const initialState = {
   nodes: [],
+  cursor: -1,
```

Comment:
this is the right default value because we 0 index the cursor?

Reply:
Exactly, yeah